app/interactors/weetInteractor.js

```diff
@@ -0,0 +1,20 @@
+const weetService = require('../services/weetsService');
+const userService = require('../services/userService');
+const errorMessages = require('../constants/errorMessages');
+const { validationError, databaseError } = require('../errors');
+const logger = require('../logger');
+
+exports.createWeet = async body => {
+  const user = await userService.findUserByEmail(body.email);
+  if (!user) throw databaseError(errorMessages.userNotFound);
+
+  const content = await weetService.getOneRandomPhrase();
+  if (content.length > 140) {
+    logger.info('Weet length error');
+    throw validationError(errorMessages.invalidWeet);
+  }
```

Comment:
This does not look like a validation error. A validation error should be resolved by endpoint consumer, and in this case is an error of the server itself.
So, I think you can truncate the obtained phrase or get a new one. what do you think?